tooling/camel-spring-boot-generator-maven-plugin/src/main/java/org/apache/camel/springboot/maven/AbstractSpringBootGenerator.java

```diff
@@ -86,8 +86,11 @@ protected String getMainDepArtifactId() {
     }
 
     protected String getMainDepGroupId() {
-        if ("camel-spring-boot-starter".equals(project.getArtifactId())) {
+        String artifactId = project.getArtifactId();
+        if ("camel-spring-boot-starter".equals(artifactId)) {
             return "org.apache.camel.springboot";
+        } else if ("camel-sap-starter".equals(artifactId)) {
```

Comment:
by renaming the groupId this can be removed too

Reply:
Not sure we should use Apache in the groupId... waiting on feedback from @cunningt